cli/common/terminal/ui.go

```diff
@@ -66,31 +65,6 @@ type Input struct {
 	Secret bool
 }
 
-// Interpret decomposes the msg and arguments into the message, style, and writer.
-func Interpret(msg string, raw ...interface{}) (string, string, io.Writer) {
```

Comment:
I had to move this to be a method because it needs access to the buffer it's going to write to as a default (but that buffer may be overwritten in this function). It reduces reusability, but I feel it's unlikely we have more that one output implementation for this project. (e.g. the CLI will always output to a command line/terminal)